src/perf/lib/ThroughputClient.cpp

```diff
@@ -47,6 +47,8 @@ ThroughputClient::ThroughputClient(
     QuicZeroMemory(&LocalIpAddr, sizeof(LocalIpAddr));
     if (Session.IsValid()) {
         Session.SetAutoCleanup();
+        Session.SetDisconnectTimeout(RPS_DEFAULT_DISCONNECT_TIMEOUT);
```

Comment:
We should either have a separate timeout for throughput, or not scope this.

Reply:
I don't understand?

Reply:
Oh, this was put in the wrong file. I will fix.

Reply:
Since this is in Throughput, but is using an RPS default. Either just change RPS_DEFAULT_DISCONNECT_TIMEOUT to PERF_DEFAULT_DISCONNECT_TIMEOUT, or make a separate one just for throughput. Would probably prefer PERF_DEFAULT though.

Reply:
Or that works too 👍